fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/wsrpc/state/SocketStateMachine.kt

```diff
@@ -128,6 +132,11 @@ object SocketStateMachine {
                         state.copy(pendingSendables = state.pendingSendables - event.sendable)
                     }
                     is Event.Stop -> handleStop(sideEffects)
+                    is Event.SwitchUrl -> {
+                        applySwitchUrlSideEffects(event.url, sideEffects)
+
+                        State.Connecting(state.url, state.attempt, state.pendingSendables)
```

Comment:
Isn't there a way to unite two Event.SwitchUrl -> {} branches? They both do exactly the same. And I see few more duplicates